pkg/server/server.go

```diff
@@ -30,7 +30,7 @@ type Config struct {
 type PluginConfig struct {
 	UseTenantInHeader               bool          `json:"useTenantInHeader,omitempty" yaml:"useTenantInHeader,omitempty"`
 	IsStreamingEnabledInDefaultPage bool          `json:"isStreamingEnabledInDefaultPage,omitempty" yaml:"isStreamingEnabledInDefaultPage,omitempty"`
-	LokiTenanLabelKey               string        `json:"lokiTenanLabelKey,omitempty" yaml:"lokiTenanLabelKey,omitempty"`
+	LokiTenantLabelKey              string        `json:"lokiTenantLabelKey,omitempty" yaml:"lokiTenantLabelKey,omitempty"`
```

Comment:
Thnx for the fix @xperimental, [here](https://github.com/openshift/logging-view-plugin/blob/main/web/src/logs.types.ts#L1) are the Config fields that the frontend reads. So it should be:
```suggestion
	AlertingRuleTenantLabelKey              string        `json:"alertingRuleTenantLabelKey,omitempty" yaml:"alertingRuleTenantLabelKey,omitempty"`
	AlertingRuleNamespaceLabelKey              string        `json:"alertingRuleNamespaceLabelKey,omitempty" yaml:"alertingRuleNamespaceLabelKey,omitempty"`
```